document_page/document_page.py

```diff
@@ -177,3 +180,14 @@ def getDiff(self, v1, v2):
                 "Revision-{}".format(v2),
                 context=True
             )
+
+    def __getattr__(self, attr):
+        """Return a dummy callabale"""
+        if attr in ['_sql', 'init', '_ids']:
+            raise AttributeError
+
+        _logger.warning(
+            "Trying to access attribute %s on document_page_history",
+            attr
+        )
+        return (lambda *a, **b: None)
```

Comment:
There is a problem in openerp, workflow will run even if modules aren't loaded during tests for example. For this reason, document_page was calling a workflow from document_page_approval which called a method that doesn't exist yet. It should be refactored differently.